trace/trace.go

```diff
@@ -165,6 +167,28 @@ func SQLSpan(
 	return sp, ctx
 }
 
+// ElasticsearchSpan starts a new elasticsearch child span with specified tags
+func ElasticsearchSpan(
```

Comment:
Line breaks. https://github.com/golang/go/wiki/CodeReviewComments#line-length

Reply:
Im shortening the name of the function and wrapping this up on the same line but other than that I cant see how it could be more dense.

Reply:
this was only about avoiding the line breaks on each argument.